scripts/scrollback-resize-guard.sh

```
@@ -0,0 +1,196 @@
+
+run "clear; for i in \$(seq 1 $NLINES); do printf 'L%04d\\n' \$i; done"
+LAST=$(printf 'L%04d' "$NLINES")
+wait_until 30 "the $NLINES numbered lines" screen_has "^$LAST\$"
+# Park the cursor above the bottom row, the way a TUI sitting in its input box
+# does. The sleep is reaped with the tmux server, never interrupted: a ctrl+c
+# would redraw a prompt over the rows under test. The PARKED marker prints
+# first, so its appearance means the park command is running.
+run "printf 'PARKED\\n'; printf '\\033[12;1H'; sleep 900"
+wait_until 15 "the parked cursor" screen_has "PARKED"
+
+snap > /tmp/dragtmux-$TAG-before.txt || fail "read-screen failed on the before snapshot"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)